modin/experimental/sql/test/test_sql.py

```diff
@@ -50,18 +51,22 @@ def test_sql_query():
 
 
 def test_sql_extension():
+    import modin.config as cfg
+
+    if cfg.StorageFormat.get() == "Hdk":
+        # HDK backend does not support DataFrame.query()
```

Comment:
Why? DataFrame.query() should be defaulting to pandas for HDK then. Also, where is `.query` called?

Reply:
This test is for DataFrame.sql() method. This method is injected by dfsql.extensions. In the HDK environment, there is no dfsql module and, thus, this test fails.

Reply:
Let's add a commend regarding this. Also, my concern is about `DataFrame.query()` in the comment above. We do not call .query() method here as I see.

Reply:
> We do not call .query() method here as I see.

It's called on line 69.

Reply:
I second @YarShev on adding a comment explaining _why_ HDK does not support `query`

Reply:
> > We do not call .query() method here as I see.
> 
> It's called on line 69.

df.sql is called there 😄 Please change `query` to `sql` in the comment.

Reply:
Sorry, you are right, the comment is not correct.